addons/l10n_be_pos_sale/static/src/js/PaymentScreen.js

```diff
@@ -16,11 +16,15 @@ patch(PaymentScreen.prototype, {
         });
     },
     toggleIsToInvoice() {
-        if (this.checkIsToInvoice()) {
+        const orderLines = this.currentOrder.get_orderlines();
+        const has_intracom_taxes = orderLines.some((line) =>
+            line.tax_ids?.some((tax) => this.pos.intracom_tax_ids.includes(tax.id))
+        );
+        if (this.checkIsToInvoice() && has_intracom_taxes) {
```

Comment:
Hi @davidmonnom, I notice the disappearance of condition `this.currentOrder.is_to_invoice()` (maybe it's deprecated?) but no use of the to_invoice variable. Any reason for that change ? I guess it's only useful for setup and to execute the toggle.
https://github.com/odoo/odoo/pull/148170/commits/9c37f42ef4e23372f5e2bdbb956b625b3a47d8e1